plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitManager.java

```diff
@@ -310,19 +311,22 @@ private static String buildSplitPath(String tableLocation, AddFileEntry addActio
     {
         // paths are relative to the table location and are RFC 2396 URIs
         // https://github.com/delta-io/delta/blob/master/PROTOCOL.md#add-file-and-remove-file
-        URI uri = URI.create(addAction.getPath());
-        String path = uri.getPath();
+        String path = addAction.getPath();
+
+        URI uri = URI.create(path);
+
+        if (!uri.isAbsolute()) {
+            path = Locations.appendPath(tableLocation, uri.getPath());
```

Comment:
static import for `Locations.appendPath`

Reply:
done